projects/js-packages/publicize-components/src/social-store/types.ts

```diff
@@ -7,6 +7,8 @@ export type SharesData = {
 	shared_posts_count: number;
 };
 
+export type ConnectionStatus = 'ok' | 'must_reauth' | 'invalid' | 'broken' | 'refresh-failed';
```

Comment:
Those are the statuses I found in `external-connections.php` - fbhepr%2Skers%2Sjcpbz%2Sjc%2Qpbagrag%2Syvo%2Srkgreany%2Qpbaarpgvbaf.cuc%3Se%3Q97ooo5qs%231203-og